### ISSUE TITLE
LPeg 1.0 Released

### ISSUE THREAD

**Issue by jack1243star** [luapower/lpeg#1]
The source is available for [download](http://www.inf.puc-rio.br/~roberto/lpeg/lpeg-1.0.0.tar.gz) now. If you have some free time could you please update the package? :smile:

Also, the short description on Github still has the version marked 0.10, perhaps it should be removed?
![default](https://cloud.githubusercontent.com/assets/2777161/10863923/eae854f0-8016-11e5-9a1b-7bcf34459546.PNG)

**Comment by capr** [comment]
Hi,

Thanks for the heads up. What's new in 1.0? Is there a changelog/announcement?

**Comment by capr** [comment]
Right, I see: http://lua-users.org/lists/lua-l/2015-09/msg00304.html

**Comment by capr** [comment]
Alright, I'll upgrade, but not immediately, I'm a bit busy at the moment. Feel free to send a PR if you can build the binaries on all platforms.

**Comment by jack1243star** [comment]
Thank you for your fast response. Unfortunately I do not own a Mac, so I cannot contribute now.

The current version is working perfectly, so do take your time. Once again thank you for your hard work making Lua more approachable!

**Comment by capr** [comment]
I hear ya. I hate it that Apple doesn't allow Mac on a VM or I would've put one up for doing automatic building and CI. I have to fix this someday because it's really ridiculous that you need actually hardware to make a damn binary.

Your best bet right now seems to be https://github.com/tpoechtrager/osxcross but I haven't used it.

PS: Thanks for the kind words, appreciated!
